packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -98,19 +105,16 @@ export default function TdpClientCanvas(props: Props) {
       };
       canvas.onmouseup = onmouseup;
       const onwheel = (e: WheelEvent) => {
-        e.preventDefault();
         onMouseWheelScroll(tdpCli, e);
       };
       canvas.onwheel = onwheel;
 
       // Key controls.
       const onkeydown = (e: KeyboardEvent) => {
-        e.preventDefault();
```

Comment:
moved `e.preventDefault()` to `useTdpClientCanvas.tsx` (which holds the default callbacks for `DesktopSession`s), because we don't want to prevent default with the `DesktopPlayer`.

Reply:
Why not? What do we need keyboard input for in the player? Play/pause controls?

Reply:
We don't _need_ it necessarily (currently we don't have keyboard controls defined, though we might do so in the future). 

The original purpose of those preventDefault's on the canvas was to ensure that what a user was doing on the remote machine wasn't affecting the browser. For example a user intending to press Windows+T on the remote machine (corresponding to cmd+T on Mac) shouldn't find themselves with a new browser tab open. However that's no longer relevant.